.travis.yml

```diff
@@ -25,10 +25,10 @@ matrix:
           sources: [ 'ubuntu-toolchain-r-test', 'llvm-toolchain-precise-3.5' ]
           packages: [ 'gdb', 'clang-3.5', 'libstdc++-4.9-dev', 'libstdc++6', 'libllvm3.4', 'xutils-dev', 'libxxf86vm-dev', 'x11proto-xf86vidmode-dev', 'mesa-utils' ]
     - os: osx
-      osx_image: beta-xcode6.3
+      osx_image: beta-xcode6.4
```

Comment:
This wasn’t quite right. See 5f7575dd8feea880fd4ab1c414e281db2ef66cf6 for the fix.